app/controllers/api/v0/swagger_blocks_extensions.rb

```diff
@@ -0,0 +1,17 @@
+module Api::V0::SwaggerBlocksExtensions
+
+  def self.included(base)
+    base.extend(ClassMethods)
+  end
+
+  module ClassMethods
+    def add_properties(*schema_names, &block)
+      schema_names.each do |schema_name|
+        swagger_schema schema_name do
+          instance_eval(&block)
+        end
+      end
+    end
+  end
```

Comment:
This stuff is already in swagger-rails: https://github.com/openstax/swagger-rails/blob/27bb0d519026707b249fde2fb101f981e508389c/lib/openstax/swagger/swagger_blocks_extensions.rb